test/benchmarks/benchmark_compare256_rle.cc

```diff
@@ -61,12 +61,12 @@ class compare256_rle: public benchmark::Fixture {
 
 BENCHMARK_COMPARE256_RLE(c, compare256_rle_c, 1);
 
-#ifdef UNALIGNED_OK
+#if BYTE_ORDER == LITTLE_ENDIAN && OPTIMAL_CMP >= 32
```

Comment:
I don't think you need to care about `OPTIMAL_CMP` for unit tests. These should just all be run.